### ISSUE TITLE
args="-n2" in draw() method causes error

### ISSUE THREAD

**Issue by mahermassoud** [pygraphviz/pygraphviz#62]
I have a dot file with the pos attribute defined for all nodes. I'd like to draw it without applying any layout. This can be done with neato and the -n2 option. I follow the [example](https://github.com/pygraphviz/pygraphviz/blob/master/examples/miles.py), miles.py from this repository and I get an error.
The method call:
`biogrid_pgv.draw("biogrid.svg",prog="neato",args="-n2")`
The error:

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-95-1425a4ad4f8d> in <module>()
      1 biogrid_pgv = pgv.AGraph("biogrid.dot")
----> 2 biogrid_pgv.draw("biogrid.svg",prog="neato",args="-n2")

/cellar/users/mahermassoud/anaconda3/lib/python3.4/site-packages/pygraphviz/agraph.py in draw(self, path, format, prog, args)
   1466             args = ' '.join([args, "-T" + format])
   1467 
-> 1468         data = self._run_prog(prog, args)
   1469 
   1470         if path is not None:

/cellar/users/mahermassoud/anaconda3/lib/python3.4/site-packages/pygraphviz/agraph.py in _run_prog(self, prog, args)
   1330 
   1331         if len(errors) > 0:
-> 1332             warnings.warn(b"".join(errors), RuntimeWarning)
   1333 
   1334         return b"".join(data)

TypeError: can't use a string pattern on a bytes-like object
```

Running the effective equivalent on the same file,
`neato -n2 -Tsvg biogrid.dot > biogrid.svg` in terminal yields expected results.
Does someone have an idea where I can look to fix this? I am using PGV 1.3rc2

**Comment by hagberg** [comment]
Does the miles.py example work for you?

**Comment by massoudTheDude** [comment]
I get compiler errors-- all relating to Strings and bytes, like:

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-6-917dc743fd44> in <module>()
    103     warnings.simplefilter('ignore', RuntimeWarning)
    104 
--> 105     G=miles_graph()
    106     print ("Loaded miles_dat.txt containing 128 cities.")
    107 

<ipython-input-6-917dc743fd44> in miles_graph()
     63     cities=[]
     64     for line in fh.readlines():
---> 65         if line.startswith("*"): # skip comments
     66             continue
     67 

TypeError: startswith first arg must be bytes or a tuple of bytes, not str
```
Could this be due to my Python version? I am running 3.4.3 with anaconda.
Thanks

**Comment by hagberg** [comment]
Possibly a version issue.  It runs with 3.4.2 though - see https://travis-ci.org/pygraphviz/pygraphviz/jobs/7159791

Are you sure you have the latest version?

**Comment by massoudTheDude** [comment]
Yes, I checked and got this:
```
$ python -V
Python 3.4.3 :: Anaconda 2.2.0 (64-bit)
```

**Comment by hagberg** [comment]
How about the pygraphviz version?

**Comment by massoudTheDude** [comment]
I'm running pygraphviz 1.3rc2

**Comment by hagberg** [comment]
I just verified that my setup (Linux) works with Python3.4.3,
pygraphviz-1.3rc2, graphviz-2.36.0.

Maybe if you post the errors we can figure it out?

**Comment by neurovertex** [comment]
Had the same issue. Adding .decode() line 1332, warnings.warn(b"".join(errors)**.decode()**, RuntimeWarning) fixed it

**Comment by hagberg** [comment]
Fixed in #94.
